src/main/java/edu/ucsd/sbrg/bigg/AnnotateDB.java

```diff
@@ -56,18 +56,23 @@ void closeConnection() {
             return annotations;
         }
 
-        String query = SELECT + "m." + COLUMN_TARGET_TERM + ", ac." + COLUMN_URLPATTERN + FROM + MAPPING_VIEW + " m, " + ADB_COLLECTION + " ac,"+ WHERE +
-                "m." + COLUMN_SOURCE_NAMESPACE + " = %s AND " + "m." + COLUMN_SOURCE_TERM + " = %s" + " AND ac." + COLUMN_NAMESPACE + " = m." + COLUMN_TARGET_NAMESPACE;
+        if(type.equals(BIGG_METABOLITE) && biggId.substring(0,2).equals("M_")) biggId = biggId.substring(2);
+        if(type.equals(BIGG_METABOLITE) && biggId.substring(0,2).equals("R_")) biggId = biggId.substring(2);
+        if(biggId.substring(biggId.length()-2,biggId.length()-1).equals("_")) biggId = biggId.substring(0,biggId.length()-2);
```

Comment:
It is not recommendable to write if clauses on one line. There should always be curly braces and the code to be executed if the condition evaluates to true should be written on the next line.

Also, It would be recommendable to define constants for the prefixes `M_` and `R_` (as well as `G_` for genes).

Reply:
Please review and merge #63 first.